hail/src/main/scala/is/hail/variant/MatrixTable.scala

```diff
@@ -61,6 +65,15 @@ object RelationalSpec {
 
     jv.extract[RelationalSpec]
   }
+
+  def readReferences(hc: HailContext, path: String): Array[ReferenceGenome] = {
+    val jv = readMetadata(hc, path)
```

Comment:
Should this also be used in Table.scala?

Reply:
Table calls into this logic also (I'm not quite sure why it's on MatrixTable.scala, but this is the generic RelationalSpec reading logic.)